test/regression/regression.conf

```diff
@@ -3,3 +3,4 @@
 shared_preload_libraries = 'pg_duckdb'
 duckdb.force_execution = true
 log_temp_files = -1
+duckdb.disabled_filesystems = ''
```

Comment:
This should not be necessary. If we need to do this in tests it means this is a bad default (at the moment). Having a default  that breaks half of pg_duckdb half of the functionality completely, even for the superuser

I think there's two things we need to do:
1. temporarily change `duckdb_disabled_filesystems` to `''` when we know we want to interact with the filesystem. Like installing duckdb extensions, or when caching a parquet file.
2. Make sure that for postgres superusers this setting is always set to `''`. They can already access files on the filesystem in various ways, so there's no point in blocking them when executing queries in DuckDB. Also they could just change the `duckdb.disabled_filesystems` setting.

Those two should probably build on top of #342